File: modulo1/segmento3/apunte6/Ejercicio5.web/ejercicio5.js
function ejercicio5 (){
	var num ;
	var max = 0;
	while(num != 0) {
		num = parseInt(prompt('Ingrese un numero (0 Finalizar): '));
		if (num >= max)
			max = num

	}
	alert('El mayor  de todos los numeros ingresados es : ' + max);
}

File: modulo1/segmento3/apunte6/Ejercicio5.java
import java.util.Scanner;

public class Ejercicio5 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

		int num , max = 0 ;

		do
			{
				System.out.print("Ingrese un numero entero positivo (0 Finalizar): ");
				num = keyboard.nextInt();
				if (num >= max){
					max = num; 
				}
			}
			while(num != 0);
			System.out.println("El mayor de todos los numeros ingresados es : " + max);
        }
}